src/main/java/org/aksw/gerbil/database/ExperimentDAOImpl.java

```diff
@@ -16,7 +16,9 @@
  */
 package org.aksw.gerbil.database;
 
+import java.io.ByteArrayOutputStream;
 import java.io.IOException;
+import java.io.ObjectOutputStream;
```

Comment:
I think this isn't needed, anymore. Can you please double check which of the inputs is actually necessary?

Reply:
Removed unnecessary Imports.